configuration/src/lib.rs

```diff
@@ -132,5 +134,13 @@ async fn find_configs_root() -> anyhow::Result<PathBuf> {
             return Ok(PathBuf::from(path_config));
         }
     }
-    anyhow::bail!("Ran out of places to find config.toml")
+
+    tracing::warn!("Config file does not exist. Creating new default.");
```

Comment:
as a operator i would appreciate the additional info log about the successful creation of the config as a follow-up to this warning, what do you think?